lib/laa_crime_schemas/structs/investment.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module LaaCrimeSchemas
+  module Structs
+    class Investment < Base
+      attribute :investment_type, Types::InvestmentType
+      attribute :description, Types::String
```

Comment:
Does description need to be a jsonb style hash as Payment has now become?

Reply:
I don't think so @mseedat-moj . This is the design:
<img width="805" alt="Screenshot 2024-03-08 at 08 37 17" src="https://github.com/ministryofjustice/laa-criminal-legal-aid-schemas/assets/34935/0f5c863c-cc99-499d-96ba-13c2937647e5">